src/gsi/write_incr.f90

```diff
@@ -202,7 +211,7 @@ subroutine write_fv3_inc_ (grd,sp_a,filename,mype_out,gfs_bundle,ibin)
     iqg = getindex(svars3d,'qg')
 
     istatus=0
-    call gsi_bundlegetpointer(gfs_bundle,'q', sub_qanl, iret); istatus=istatus+iret
+    call gsi_bundlegetpointer(svalinc(ibin2),'q', sub_qanl, iret); istatus=istatus+iret
```

Comment:
good catch on this, no idea why this one is different from the rest... does this have any implcations for current GFSv16?

Reply:
@jderber-NOAA   I am a little confused here.  In the following around line 313 (in this program), there is 
`  ! compute q increment
    sub_q = sub_qanl(:,:,:) - ges_q1(:,:,:,ibin)
`
So, seems to me, the original code intentionally gets analysis q in full at the above (line 205) .   
I am wondering why the original treatment is used .  However, if the change of line 214 is used, the following line 313 should also be changed too?

Reply:
All regression tests passed and test of code was fine.  I believe this is ready for re-review.